terraform-post-packer/backend.tf

```diff
@@ -5,6 +5,6 @@ terraform {
     dynamodb_table = "terraform-state-lock"
     profile        = "cool-terraform-backend"
     region         = "us-east-1"
-    key            = "freeipa-server-packer/terraform-test-user.tfstate"
+    key            = "skeleton-packer/terraform-post-packer.tfstate"
```

Comment:
This key should be: `freeipa-server-packer/terraform-post-packer.tfstate`

Reply:
Please have a look at commit 8ca1ace and let me know if I caught everything.

Reply:
Yes, that all looks correct to me- good catch on the providers and caller identity!